Cargo.toml

```diff
@@ -10,7 +10,7 @@ authors = ["Snirk Immington <snirk.immington@gmail.com>", "Timidger <apragmaticp
 build = "build.rs"
 
 [dependencies]
-rustwlc = { version = "0.6.4", features = ["wlc-wayland"] }
+rustwlc = { version = "0.7.0", features = ["wlc-wayland"], git = "https://github.com/Idleless/rust-wlc" }
```

Comment:
I have published this version now on crates.io, please remove this git link.

Reply:
I have also updated dummy-rustwlc. Please update the `Cargo.toml` listing to point to version 0.7.0 for dummy-rustwlc and the build should now pass.